scripts/signing_secure_boot_dev.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
```

Comment:
DEV vs PROD are actually key types right? Why to mandate the signing method as DEV vs PROD ?
Why not DEV keys are external signing server or Production signing is based on local public key. Not every vendor would have external signing server.
By default SONiC should support sbsign method and signing private keys to be available within build system for both production and development purpose and additional hook to provide for vendor specific signing where vendor can implement their choice of signing or external signing service. The vendor specific signing should be kept out side of build hook and linked via build configuration.

Reply:
This is exactly why we implemented this feature the way we showcased here.
In our dev signing script, we are using a local script since we are going to assume a dev environment functions as an environment that has everything at hand - a build server, dev keys, dev certs all accessible. Also the reason why we are using it as a folder/location access to the keys (assuming they reside in some FS location accessible by the build server)
For production environment - we have no idea what each environment requests, it might be the same as dev (in this case we can just copy the script) or it could be that there is a signing server that resides in some secure location where you have to call specific API calls with a certain api token to send requests to it. 
Therefor - production value signing is open for the customer and dev signing is given here in full.
We provided a placeholder for the build hook to be called.